### ISSUE TITLE
Multi project solution - deploy in docker containers

### ISSUE THREAD

**Issue by betimd** [awslabs/aws-blog-net-exploring-aspnet-core#1]
Was a bit confuse how to build docker image in a VS solution with several projects, like a web project with two class library projects (like data access, business logic, or external api callings).
So building a sample like this and deploying in docker containers (EC2 CS) would be helpful for several scenarios.

**Comment by normj** [comment]
Covering docker is definitely something I want to cover. I want to write up about using docker for both Elastic Beanstalk and EC2 Container Service.

**Comment by betimd** [comment]
And covering image source would be very helpful, like ec2 cs registry and docker hub as well.

**Comment by normj** [comment]
Yes, definitely want cover our registry service. Thanks for the feedback!

**Comment by betimd** [comment]
Actually I'm using aspnet vNext & AWS in building our company infrastructure and products and I want to contribute in this journey not just in feedback but code / samples as well. Are you going to open this repo for PRs or you'll keep just inside aws team?

**Comment by normj** [comment]
I'm definitely open to pull requests. Just keep in mind any code submitted would go under the Apache 2 license.
